Dockerfile

```diff
@@ -17,4 +17,10 @@ COPY ./src/api /usr/src/api
 
 WORKDIR /usr/src/api
 
-ENTRYPOINT ["gunicorn", "-w", "4", "--log-syslog", "--bind", "0.0.0.0:5000", "main:app"]
+ENTRYPOINT ["/usr/local/bin/gunicorn"]
+
+CMD ["--workers", "4", \
+     "--timeout", "60", \
```

Comment:
True. We can also override this value within model-my-watershed by modifying the upstart job config for testing.